.restyled.yaml

```diff
@@ -3,3 +3,7 @@
     include:
       - downgrade
     image: restyled/restyler-shfmt:v3.0.1
+    
+- prettier-markdown:
+    include:
+      - README.md
```

Comment:
:+1:

How bad is it if you let prettier-markdown run on the manpage source? If it makes a crazy diff, we can hold off for now, but if it's not too bad I think it'd be good to wrap Restyled on that too, or any markdown in the repo (e.g. the default `include`).

Reply:
Let's give it a try, committing it now.

Reply:
Hmm, for the manpage source markdown it seems to interfere with some of the ultimate manpage output. I would prefer if we exclude this from restyled since this manpage markdown is not exactly something we present but is more of a tool for us to generate the manpage.